<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionToolPopup</class>
 <widget class="QWidget" name="SelectionToolPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QToolButton" name="lasso_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select structure (with lasso)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../sketcher.qrc">
       <normaloff>:/icons/select_lasso.svg</normaloff>:/icons/select_lasso.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="rect_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select structure (with square marquee)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../sketcher.qrc">
       <normaloff>:/icons/select_square.svg</normaloff>:/icons/select_square.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="ellipse_btn">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Select structure (with ellipse marquee)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../sketcher.qrc">
       <normaloff>:/icons/select_ellipse.svg</normaloff>:/icons/select_ellipse.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">group</string>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="group"/>
 </buttongroups>
</ui>
